trestle/oscal/common.py

```diff
@@ -1626,6 +1626,7 @@ class Config:
     values: Optional[List[constr(regex=r'^\S(.*\S)?$')]] = Field(None)
     select: Optional[ParameterSelection] = None
     remarks: Optional[str] = None
+    param_value_origin: Optional[str] = None
```

Comment:
This is wrong. We are not adding a field directly to param origin in OSCAL. We are just adding a property. So OSCAL schema should not be changed.